src/beamlime/services/fake_detectors.py

```diff
@@ -173,16 +167,18 @@ def serialize_detector_events_to_ev44(
 
 
 def run_service(*, instrument: str, log_level: int = logging.INFO) -> NoReturn:
-    service_name = f'{instrument}_fake_producer'
     kafka_config = load_config(namespace=config_names.kafka_upstream)
-    source = FakeDetectorSource(instrument=instrument)
     serializer = serialize_detector_events_to_ev44
-    processor = StreamProcessor(
-        source=source,
-        sink=KafkaSink(kafka_config=kafka_config, serializer=serializer),
+    builder = DataServiceBuilder(
```

Comment:
All the `fake_*` services were previously not using `DataServiceBuilder`. Now they can, by adding a new method to the builder. This is in a sense unrelated to the main change.